.github/dependabot.yml

```diff
@@ -0,0 +1,13 @@
+# Configuring dependabot.yml: https://docs.github.com/en/github/administering-a-repository/configuration-options-for-dependency-updates
+
+version: 2
+updates:
+  - package-ecosystem: "npm"
```

Comment:
@chern Does it make a difference we are using yarn? Will it create pull requests that update our yarn.lock file?

Reply:
@rlorenzo Yarn isn't listed explicitly as a package-ecosystem (https://docs.github.com/en/github/administering-a-repository/configuration-options-for-dependency-updates#package-ecosystem), but Dependabot's own website says it supports Yarn lockfiles: https://dependabot.com/javascript/